frameworks/storage.rst

```diff
@@ -0,0 +1,186 @@
+.. _framework-storage:
+
+=================
+Storage Framework
+=================
+
+The storage framework provides a plugin-based system for short- and long-term storage of
+data, accessible from Zeek script-land. This is not packet data itself, but data artifacts
+generated from the packet data. It has interchangeable asynchronous and synchronous
+modes.
+
+This chapter gives an overview of the storage framework, plus examples of using it. For
+more examples, see the test cases in ``testing/btest/scripts/base/frameworks/storage`` and
+an example storage plugin test in ``testing/btest/plugin/storage-src``.
+
+Terminology
+===========
+
+Zeek's storage framework uses two main components:
+
+  Backend
+    A backend plugin provides access to a storage system, called a ``store`` in the rest
```

Comment:
Total nitpickery: The doc doesn't use "store" in that manner all that much (maybe twice), so maybe just ditch the note about "store"? The examples also assign the result of `open_backend()` to a variable called `backend` rather than `store`, so that just seems a bit inconsistent. (personally like backend and/or backend instance more).